UI/Selenium/Steps/HearingRoomSteps.cs

```diff
@@ -209,5 +209,50 @@ public void WhenTheJudgeCanOpenAndCloseTheChatPanel()
                 }
             }
         }
+
+        [When(@"the judge can open and close the chat panel")]
+        public void WhenTheJudgeCanOpenAndCloseTheChatPanel()
+        {
+            var judge = _hearing.Participant.Where(a => a.Id.ToLower().Contains("judge")).FirstOrDefault();
```

Comment:
What would happen when a hearing has two or more judges whose roles might be panel members or wingers?